csv-data-management/extract_data_from_csv.py

```diff
@@ -456,11 +499,11 @@ def _ds_filter(params, ds):
         if input_type == 'On Values Matching a Regex':
             reg = params.get('filter')
             columnName = params.get('filterColumnName')
-            df = df[df[columnName].str.match(reg) == True]
+            df = df.filter(pl.col(columnName).str.contains(reg))
```

Comment:
while we are at it, could you cleanup the variable reinitialization at line 496?

Reply:
> while we are at it, could you cleanup the variable reinitialization at line 496?
Yes, Done.